.github/workflows/chimney-script-config-change-pr.yaml

```diff
@@ -8,3 +8,479 @@ jobs:
     runs-on: arc-runner
     steps:
       - uses: actions/checkout@v3
+
+      - name: "Clone Blobber Repo"
+        uses: actions/checkout@v3
+        with:
+          ref: staging
+          repository: 0chain/blobber
+          fetch-depth: 1
+          path: ./blobber
+
+      - name: "Install yq"
+        run: |
+          sudo apt update && sudo apt install wget -y
+          sudo wget https://github.com/mikefarah/yq/releases/download/v4.31.1/yq_linux_amd64 -O /usr/bin/yq && sudo chmod +x /usr/bin/yq
+          yq --version
+
+      - name: "Update 0chain_blobber.yaml"
+        run: |
+          yq -i '.read_price = "${READ_PRICE}"' blobber/config/0chain_blobber.yaml
+          yq -i '.write_price = "${WRITE_PRICE}"' blobber/config/0chain_blobber.yaml
+          yq -i '.delegate_wallet = "${DELEGATE_WALLET}"' blobber/config/0chain_blobber.yaml
+          yq -i '.num_delegates = "${NO_OF_DELEGATES}"' blobber/config/0chain_blobber.yaml
+          yq -i '.service_charge = "${SERVICE_CHARGE}"' blobber/config/0chain_blobber.yaml
+          yq -i '.block_worker = "${BLOCK_WORKER_URL}"' blobber/config/0chain_blobber.yaml
+          yq -i '.admin.username = "${GF_ADMIN_USER}"' blobber/config/0chain_blobber.yaml
+          yq -i '.admin.password = "${GF_ADMIN_PASSWORD}"' blobber/config/0chain_blobber.yaml
+
+      - name: "Update 0chain_validator.yaml"
+        run: |
+          yq -i '.delegate_wallet = "${DELEGATE_WALLET}"' blobber/config/0chain_validator.yaml
+          yq -i '.service_charge = "${SERVICE_CHARGE}"' blobber/config/0chain_validator.yaml
+          yq -i '.block_worker = "${BLOCK_WORKER_URL}"' blobber/config/0chain_validator.yaml
+
+      - name: "Copy 0chain_blobber.yaml and 0chain_validator.yaml in zcnwebappscripts repo"
```

Comment:
rather than harcoding the script, we should dynamically fetch the current script. So that we don't have a make changes at 2 places.